index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Joleen Tsai</title>
+</head>
+<link rel="stylesheet" href="assets/css/reset.css">
+<link rel="stylesheet" href="assets/css/style.css">
+
+<body>
+    <header class="header">
+        <div class="namebox">Joleen Tsai</div>
+        <div class="topbuttons">
+            <a href="index.html" class="topbuttons">About</a>
+            <a href="portfolio.html" class="topbuttons">Portfolio</a>
+            <a href="contact.html" class="topbuttons">Contact</a>
+        </div>
+    </header>
+    <div class="container">
+        <div class="indexcontent">
+            <h1 class="title">About Me</h1>
+            <img src="assets/images/Headshot.JPG" img class="headshot" alt="Headshot">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet pariatur, a atque architecto explicabo iure corrupti.
```

Comment:
Since this is so long, I would put the `<p>` tag on another line that all the text content.